mxd/backend-service/src/main/java/org/eclipse/mxd/entity/Transfer.java

```diff
@@ -0,0 +1,128 @@
+
+/*******************************************************************************
+ * Copyright (c) 2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *    Contributors:Ravinder Kumar
+ *    Backend-API and implementation
+ *
+ ******************************************************************************/
+
+
+package org.eclipse.mxd.entity;
+
+import jakarta.persistence.*;
+
+import java.util.Date;
+
+@Entity
+public class Transfer {
+
+    @Id
```

Comment:
Primary key will be `transfer id`, no need of auto generated seq key.

Reply:
As suggested transferId is a primary key and auto generated seq key is removed